interfaces/builtin/mir.go

```diff
@@ -49,6 +49,9 @@ capability sys_tty_config,
 
 /{dev,run}/shm/\#* mrw,
 /run/mir_socket rw,
+/run/user/[0-9]*/mir_socket rw,
```

Comment:
I'm surprised we didn't notice this access before since we allow it in mirConnectedPlugAppArmor.

Reply:
Yep, I found that funny too